src/test/java/org/jabref/logic/importer/ACMPortalParserTest.java

```diff
@@ -0,0 +1,111 @@
+
+    @BeforeEach
+    void setUp() throws URISyntaxException, MalformedURLException {
+        parser = new ACMPortalParser();
+        searchUrl = new URIBuilder("https://dl.acm.org/action/doSearch?AllField=" + searchDoi).build().toURL();
+        searchEntryList = new ArrayList<>();
+
+        BibEntry expected = new BibEntry(StandardEntryType.Article);
+
+        expected.setField(StandardField.AUTHOR, "Tobias Olsson and Morgan Ericsson and Anna Wingkvist");
+        expected.setField(StandardField.YEAR, "2017");
+        expected.setField(StandardField.MONTH, "9");
+        expected.setField(StandardField.DAY, "11");
+        expected.setField(StandardField.ABSTRACT, "The open source application JabRef has existed since 2003. In 2015, the developers decided to make an architectural refactoring as continued development was deemed too demanding. The developers also introduced Static Architecture Conformance Checking (SACC) to prevent violations to the intended architecture. Measurements mined from source code repositories such as code churn and code ownership has been linked to several problems, for example fault proneness, security vulnerabilities, code smells, and degraded maintainability. The root cause of such problems can be architectural. To determine the impact of the refactoring of JabRef, we measure the code churn and code ownership before and after the refactoring and find that large files with violations had a significantly higher code churn than large files without violations before the refactoring. After the refactoring, the files that had violations show a more normal code churn. We find no such effect on code ownership. We conclude that files that contain violations detectable by SACC methods are connected to higher than normal code churn.");
```

Comment:
Abstracts might be copyrighted, so just remove them for the test (e.g set the field to empty or remove it before comparing)

Reply:
fixed.